src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/SplashScreenPlatform.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Uno.Toolkit.UI;
+
+[Flags]
+public enum SplashScreenPlatform
+{
+	None,
+	Android = 1,
+	iOS = Android << 1,
+	Windows = iOS << 1,
+	WebAssembly = Windows << 1,
+	Skia = WebAssembly << 1,
+	All = Android | iOS | Windows | WebAssembly | Skia
```

Comment:
```suggestion
	iOS = 1 << 1,
	Windows = 1 << 2,
	WebAssembly = 1 << 3,
	Skia = 1 << 4,
	All = Android | iOS | Windows | WebAssembly | Skia
```

Reply:
I don't like this change as it make it hard to add/change, although I guess you could argue that it would be a breaking change (ie changing the underlying flag value)